archive-editorial.php

```diff
@@ -35,7 +35,7 @@
     the_post();
 ?>
 
-    <?php the_post_thumbnail('slide', array('data-id' => $post->ID, 'class' => 'archive-image')); ?>
+    <?php the_post_thumbnail( 'full', array('data-id' => $post->ID, 'class' => 'archive-image') ); ?>
```

Comment:
well yeah if we gonna do a larger responsive breakpoint then the value set dont matter much. If you want to set a height then I'd choose a ratio and add a image size `col6-2to3` or something like that